portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,8 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function addGreeting(){
+    fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('quote-container').innerText = quote;
+    });
+}
```

Comment:
nit: please add an empty line at the end of the file. Here's an interesting read on why this is a convention in the software development world: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file